### PR TITLE
Feature/#6 user profile

### LINKED ISSUE #6: Create user profile view
_From @bajiat on February 19, 2016 12:19_

Create basic user profile view. Determine important information for organization members.

Definition of done
---
* [x] Relevant user profile elements have been defined on the basis of user requirement document.
* [x] Fields have been added to user profile schema.
* [x] User profile view exists.

_Copied from original issue: Dipor/platform#13_

### PR DESCRIPTION
**Pull request by elnzv**
Closes #6 

# Proposed changes
- User profile schema with First and Last names
- /profile route with basic panel and fields with AutoForm
- sAlert package to show notifications to user
- Link to user profile from navbar

@Digipalvelutehdas/dipor-team please review.

### CODE REVIEW

**Comment by jykae** [comment]
Reviewing..

**Comment by jykae** [comment]
Ok, this misses still schema and inline edit. Good work adding packages and routing. Easy to continue.
".packages" and ".versions" are probably the only thing conflicting here with develop.
Could you add then "Proposed changes" when this is closer completion?

profile

File: docs/en.i18n.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
   "mainLayoutNavigation-home": "Home",
   "mainLayoutNavigation-organizations": "Organizations",
-  "allOrganizationsView-Organizations": "Organizations"
-}
\ No newline at end of file
+  "allOrganizationsView-Organizations": "Organizations",
+  "profile_Header": "Profile",
```
Comment (jykae) [inline docs/en.i18n.json]: Decide whether to use - or _ 
Check which one OneSky uses, if we choose that as translation platform.

Comment (brylie) [inline docs/en.i18n.json]: We are using hyphens.

File: .meteor/versions [hunk 2]
```diff
@@ -60,6 +62,7 @@ minifiers@1.1.7
 minimongo@1.0.10
 mobile-experience@1.0.1
 mobile-status-bar@1.0.6
+momentjs:moment@2.10.6
```
Comment (jykae) [inline .meteor/versions]: Good utility addition as part of this task.

**Comment by jykae** [comment]
Good work

**Pull request merged**